### PR TITLE
Benchmark with 10M ML dataset and automatic data download

### PR DESCRIPTION
**Pull request by btwardow**
I change a little bit your benchmark to:
- download 10M ML dataset & automagicaly prepare train/test files 
- remove data files from the repository

### CODE REVIEW

File: benchmarks/fm_jl_benchmark.jl [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
     (X_train, y_train) = @time read_libsvm(train_filename)
     info("Train dim: $(size(X_train))")
     (X_test, y_test) = @time read_libsvm(test_filename)
     info("Test dim: $(size(X_test))")
 
-    info("Test ML-100K - Training model...")
+    info("Training model...")
     fm = @time train(X_train, y_train, 
-        method = Methods.sgd(num_epochs = UInt(10), alpha = 0.1),
+        method = Methods.sgd(num_epochs = Int(10), alpha = 0.1),
```
Comment (Hydrotoast) [inline benchmarks/fm_jl_benchmark.jl:19]: The `Int` here is unnecessary since the default type of the `10` literal is `Int`, but this can be removed later

File: benchmarks/prepare_data.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import sys
+
```
Comment (Hydrotoast) [inline benchmarks/prepare_data.py:2]: Just being pedantic: PEP8 recommends 2 newlines between imports and code.

File: benchmarks/run_benchmarks.sh [hunk 3]
```diff
@@ -3,9 +3,30 @@
 # Change to the directory of the current script
 cd "$(dirname "$0")"
 
+# download MovieLense datasets
+TRAIN_FILENAME="data/ml-10m.train.libfm"
+TEST_FILENAME="data/ml-10m.test.libfm"
+
+if [ ! -f "$TRAIN_FILENAME" ] || [ ! -f "$TEST_FILENAME" ]
```
Comment (Hydrotoast) [inline benchmarks/run_benchmarks.sh:10]: I haven't tried this edge case, but if exactly one of the two files exists, should we need to delete `rm` the entire data directory `data` to avoid any overwriting  error messages?

File: benchmarks/prepare_data.py (hunk 2 above)
Comment (btwardow) [inline benchmarks/prepare_data.py:2]: Hmm...Your're right. However, the PyMode vim plugins claiming after `:PymodeLintAuto' :

> Auto Pep8 done. 

:)

File: benchmarks/run_benchmarks.sh (hunk 3 above)
Comment (Hydrotoast) [inline benchmarks/run_benchmarks.sh:10]: This turns out to be ok.
